test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -281,22 +281,49 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			step( 'Can see the Earn blocks', async function () {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.openBlockInserterAndSearch( 'earn' );
+				const shownItems = await gEditorComponent.getShownBlockInserterItems();
+
 				assert.strictEqual(
-					await gEditorComponent.isBlockCategoryPresent( 'Earn' ),
-					true,
-					'Earn (Jetpack) blocks are not present'
+					[
```

Comment:
> Before it was just checking if the actual DOM element for the category tab was present. Category is not rendered as part of the inserter search UI anymore, though (see https://github.com/Automattic/wp-calypso/pull/47533/files#r525642380).
> 
> We can either:
> 
>     Check for the actual blocks. This is what I did for now;
>     Get rid of those tests?
> 
> Also, we I decided to use deepEqual in order to get sort order testing for free. If that's overkill, we can use sameMembers.
> 
> Thoughts?

I momentarily forgot that this should be compatible with GB < 9.4 as well. I initially thought about having a condition that tried to find the tab element unique to 9.3, then run the previous assertion, and if not, run the one for 9.4. I decided to simplify further and make it a more shallow check, that actually doesn't rely on the order and only checks that the elements are present.

This means it won't test the sorting order and will not fail if additional blocks are shown (which happens on 9..3, see the screenshot comparison below). I think it also better satisfies the test description, too. Let me know your thoughts.

Reply:
Also, I couldn't find an assertion method in `assert` that would assert that a array included a subset, so I had to improvise. Let me know if I mssed something. cc @bsessions85

Reply:
Are we using Jest to run these? 'cause Jest has [`expect.arrayContainingArray`](https://jestjs.io/docs/en/expect#expectarraycontainingarray).

If we're _not_ using Jest, maybe we should consider migrating to it (as a follow-up)? I think Gutenberg is using it for its e2e tests, no?

Reply:
We're using Mocha. I thought we were using Chai too, but it appears it's not the case. I think migrating to Jest makes sense as a follow up issue. I'd like to know @bsessions85 thoughts on that, too.

Reply:
I've looked in to moving to jest, but the way our tests are set up, I couldn't get the the full "describe" block to re-run, just the failed steps, which didn't help. I could get close by hacking some packages, but it was going to be a maintenance problem. I'll look in to it again as its been about a year. We could look at using Chai with Mocha as well for now, even if it is just in this file to start

Reply:
Cycling back to this:

> I momentarily forgot that this should be compatible with GB < 9.4 as well.

Does it, though? If we land this PR briefly after enabling 9.4 in production, we don't necessarily need backwards-compat, do we? Or is that related to the block upgrade/migration tests?

Reply:
Ideally, but not strictly necessary. We can indeed land it after we ship 9.4.